README.md

```diff
@@ -17,31 +17,32 @@ Run `make build`.
 
 ## Testing
 
-Run `make test` to run all unit and integration tests, as well as an acceptance test. To make integration and acceptance
-tests pass, set the Oracle database URL to the environment variables as an `ORACLE_URL`.
+Run `make test` to run all unit and integration tests, as well as an acceptance test. To pass the integration and
+acceptance tests, set the Oracle database URL to the environment variables as an `ORACLE_URL`.
 
 ## Source
 
 The Oracle Source connects to the database using the provided `url` and starts creating records for each table row and
-each change detected. The first time Source runs, it creates an additional table and a trigger to track changes in
-the `table` and launches Snapshot mode. Then, when all the records have been read, Source switches to CDC mode. More
-information [inside the Change Data Capture section](#change-data-capture).
+each detected change. The first time Source runs, it makes a snapshot, creates a tracking table, and a trigger to track
```

Comment:
please add link from oracle to snaphsot for understanding what is it

Reply:
Done.